### ISSUE TITLE
Restrict which tokens can be accepted by these projects

### ISSUE THREAD

**Issue by WhyldWanderer** [Giveth/giveth-dapps-v2#2733]
We have added some projects to support the relief efforts in Turkey. 
These projects can only accept certain tokens. 
I have outlined which tokens each one can accept in this spreadsheet.
https://docs.google.com/spreadsheets/d/1GtBNwesVl_Cxr4IYeX1W7t5BHJBmQSu1WBFda35g0ag/edit#gid=0

We need to restrict the list of tokens that these projects can accept before listing and verifying them.

**Comment by mohammadranjbarz** [comment]
@MoeNick  @WhyldWanderer 
In our current architecture we have some organization, the organization specifies the list of accepted tokens and accepted chains, so each project that is connected to those organization follow the organization's specs, now I looked into the sheet and knew that we need 4 different token list for those projects, so the only way we can do without any development is creating four organization and assign projects to those organization

@CarlosQ96  WDYT?

**Comment by CarlosQ96** [comment]
Yeah I agree, else we would have to write workarounds with code.

**Comment by WhyldWanderer** [comment]
Hey @CarlosQ96 & @mohammadranjbarz - just checking in on the status of this issue. I can see that you have listed the projects but they are unable to accept donations and have no tokens on their token list. 

I would love to start the outreach to Turkey Relief DAO.... when do you think these projects will be able to accept donations in the related tokens?

**Comment by WhyldWanderer** [comment]
@MoeNick you can see the context in this issue... 
https://github.com/Giveth/giveth-planning/issues/1342

I have given you access to the sheet now. I don't see this as adding any features.. its just a matter of making it so that these projects can only accept certain tokens. 

It is not necessary for QF launch... it will be more relevant for devconnect in November but we have a lot of correspondence to do with other orgs before then and I cannot start the correspondence until these projects are live and collecting donations.

**Comment by MoeNick** [comment]
We can move with @mohammadranjbarz idea with adding another organization only if the token list is the same for all of them. Is it possible @WhyldWanderer

**Comment by MoeNick** [comment]
@mohammadranjbarz we need 
2 organization 
1) TurkeyProjects-Set1- with this token list:  ETH and USDT on mainnet
2) TurkeyProjects-Set1 -  with this token list: ETH, USDC, USDT, TRYC on mainnet

For more context pls take a look: https://docs.google.com/spreadsheets/d/1GtBNwesVl_Cxr4IYeX1W7t5BHJBmQSu1WBFda35g0ag/edit#gid=0

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz we need 2 organization
> 
> 1. TurkeyProjects-Set1- with this token list:  ETH and USDT on mainnet
> 2. TurkeyProjects-Set2 -  with this token list: ETH, USDC, USDT, TRYC on mainnet
> 
> For more context pls take a look: https://docs.google.com/spreadsheets/d/1GtBNwesVl_Cxr4IYeX1W7t5BHJBmQSu1WBFda35g0ag/edit#gid=0
> 
> and we need to have these 2 projects in TurkeyProjects-Set2
> 
> * Needs Map (İhtiyaç Haritası)
> * Foundation For The Support of Women’s Work
> 
> And have the rest in G-Sheet be part of TurkeyProjects-Set1

@WhyldWanderer  I did it you can check it in admin-bro ( but you can't test the donations because they don't have recipient address so you can't see their accepted token but in chrome network logs it was ok)

So you just need to change project's organization to **TurkeyProjects-Set1** or **TurkeyProjects-Set2** as @MoeNick  wanted.

PS: We don't have **TRYC** token
PS2: Now you cant connect tokens to organizations from admin panel page, so if you want to change the tokens of organization you should tell me to do it)

**Comment by MoeNick** [comment]
@WhyldWanderer I suggest we add TRYC token, @mohammadranjbarz whats' the blocker for adding this one?
@maryjaf can you take a look and test, please?

**Comment by mohammadranjbarz** [comment]
> @WhyldWanderer I suggest we add TRYC token, @mohammadranjbarz whats' the blocker for adding this one? @maryjaf can you take a look and test, please?
- [ ] Modify monoswap to support this token
- [ ] Write a migration
- [ ] Add some test cases to make sure it works

I think we can create another issue for that purpose

ؤیی

**Comment by maryjaf** [comment]
By taping on **Donate**, the below modal is shown and I couldn't donate to these projects
https://giveth.io/donate/community-volunteers-foundation
<img width="1419" alt="image" src="https://github.com/Giveth/giveth-dapps-v2/assets/111529185/cf5bf6ed-a59a-4679-a2b4-5c858251feb6">
@mohammadranjbarz

**Comment by WhyldWanderer** [comment]
@maryjaf the addresses have been added... thanks!

**Comment by maryjaf** [comment]
Thanks @mohammadranjbarz The displayed tokens are correct for all projects on Mainnet except TRYC which has not been added yet

According to the list, this project https://giveth.io/project/needs-map-0 should have an address on Polygon, but it doesn't @WhyldWanderer

**Comment by WhyldWanderer** [comment]
It would have been nice to include the polygon address, however it would mean starting a whole new organization for only one of the projects to have a unique token list. So we opted to just have mainnet available.
